src/main/java/seedu/address/model/log/AppointmentDate.java

```diff
@@ -18,16 +22,19 @@ public class AppointmentDate {
      * @param date A valid LocalDate.
      */
     public AppointmentDate(LocalDate date) {
```

Comment:
catches null date entry and returning message constraint, instead of throwing NULLEXCEPTION error